tests/unit/commands/local/lib/test_local_lambda.py

```diff
@@ -260,6 +263,28 @@ def test_must_resolve_absolute_codeuri(self, codeuri):
         self.assertEquals(expected, actual)
         self.assertTrue(os.path.isabs(actual), "Result must be an absolute path")
 
+    @patch("samcli.local.lambdafn.utils.os")
+    def test_must_resolve_windows_path_to_unix_paths(self, os_mock):
+
+        os_mock.name = "nt"
+        with mock.patch("os.path", ntpath):
+            ntpath.isabs = lambda path: False
```

Comment:
ooo... this is interesting, never considered this a way to mock functions out before!